docs/main.6bb055d5.iframe.bundle.js

```diff
@@ -1 +1 @@
e?this.effectiveDateType=enums.c.FUTURE_EFFECTIVE:!1===this.isFutureEffective?this.effectiveDateType=enums.c.IMMEDIATE:this.effectiveDateType=null;const effectiveDate=this.effectiveDateTime.effectiveDate,date=effectiveDate&&new Date(effectiveDate);if(date){let hour=date.getHours();const minute=date.getMinutes(),period=hour<12?PeriodTypes.AM:PeriodTypes.PM;hour>12?hour-=12:0===hour&&(hour=12),this.dateText=this.dateToYyyyMmDd(date),this.selectHour=[hour.toString()],this.selectMinute=[minute.toString().padStart(2,"0")],this.selectPeriod=period}}async constructAndUpdate(){await vue_esm.default.nextTick();const isDateValid=this.$refs.datePickerRef.validateForm(),isTimeValid=this.$refs.form.validate();if(isDateValid&&isTimeValid&&this.selectHour.length&&this.selectMinute.length){const year=+this.dateText.slice(0,4),month=+this.dateText.slice(5,7)-1,date=+this.dateText.slice(8,10);let hours=+this.selectHour;const minutes=+this.selectMinute;this.selectPeriod===PeriodTypes.AM&&12==+this.selectHour&&(hours=0),this.selectPeriod===PeriodTypes.PM&&12!=+this.selectHour&&(hours+=12);const dateTime=this.createUtcDate(year,month,date,hours,minutes);this.emitEffectiveDate(dateTime)}this.emitValid()}onCurrentJsDateChanged(val){if(val){const minDate=new Date;minDate.setTime(val.getTime()+60*this.MIN_DIFF_MINUTES*1e3),this.minDate=minDate;const maxDate=new Date;maxDate.setTime(val.getTime()+24*this.MAX_DIFF_DAYS*60*60*1e3),this.maxDate=maxDate,this.emitValid()}}onDatePickerChanged(val){this.dateText=val}onDateTextChanged(val){this.isFutureEffective&&this.constructAndUpdate()}onSelectHourChanged(val){this.isFutureEffective&&this.constructAndUpdate()}onSelectMinuteChanged(val){this.isFutureEffective&&this.constructAndUpdate()}onSelectPeriodChanged(val){this.isFutureEffective&&this.constructAndUpdate()}onEffectiveDateTypeChanged(val){this.isImmediate=val===enums.c.IMMEDIATE,this.isFutureEffective=val===enums.c.FUTURE_EFFECTIVE,this.isImmediate&&(this.datePicker="",this.dateText="",this.selectHour=[],this.selectMinute=[],this.selectPeriod=PeriodTypes.AM,this.$refs.datePickerRef.clearDate()),this.emitIsFutureEffective(this.isFutureEffective),this.emitEffectiveDate(null),this.emitValid()}emitIsFutureEffective(val){}emitEffectiveDate(val){}async emitValid(){const validDateText=!this.isFutureEffective||!!this.dateText;await vue_esm.default.nextTick();const isDateValid=this.$refs.datePickerRef.validateForm(),isTimeValid=this.$refs.form.validate();return this.isImmediate||!!this.effectiveDateType&&isDateValid&&isTimeValid&&!!this.selectHour.length&&!!this.selectMinute.length&&!this.isUnderTime&&!this.isOverTime&&validDateText}};Object(tslib_es6.a)([Object(lib.d)({default:!1})],EffectiveDateTimevue_type_script_lang_ts_EffectiveDateTime.prototype,"parseInitial",void 0),Object(tslib_es6.a)([Object(lib.d)()],EffectiveDateTimevue_type_script_lang_ts_EffectiveDateTime.prototype,"currentJsDate",void 0),Object(tslib_es6.a)([Object(lib.d)()],EffectiveDateTimevue_type_script_lang_ts_Ef
```

Comment:
Is this file auto-generated? It's not possible to review what's changed here.

Reply:
Yes, this file is auto-generated as a result of step 10 in the workflow readme.

I usually just check that the changes sort-of match the code changes. I would expect a few more files to change if I had updated a component. If there are a ton of changed Storybook files then maybe something went wrong.